### PR TITLE
MTE-5854: Use international dates from Drupal

### PR DESCRIPTION
**Pull request by WidgetsBurritos**
JIRA: https://jira.rax.io/browse/MTE-5854

This PR adds support for international dates (i.e. basically just defers date processing to drupal). I also made a modification to the dev experience. In the bottom right-hand corner of the page, you get some helper links to get you directly into the alternate views:

image

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4007,55 +4007,6 @@
...
-          "version": "6.1.0",
-          "resolved": "https://registry.npmjs.org/supports-color/-/supports-color-6.1.0.tgz",
-          "integrity": "sha512-qe1jfm1Mg7Nq/NSh6XE24gPXROEVsWHxC1LIx//XNlD9iw7YZQGjZNjYN7xGaEG6iKdA8EtNFW6R0gjnVXp+wQ==",
-          "dev": true,
-          "requires": {
-            "has-flag": "^3.0.0"
-          }
-        }
-      }
-    },
```
Comment (WidgetsBurritos) [inline package-lock.json]: Oh I need to drop this file.  Will push up again.

**Comment by WidgetsBurritos** [comment]
**[AWS]**

Preview URL: https://mte-5854-show-international-date.solve.pr.rakr.net/solve

Login Url: https://mte-5854-show-international-date.solve.pr.rakr.net/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5854-show-international-date.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5854-show-international-date.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

**Comment by WidgetsBurritos** [comment]
**[AWS]**

Preview URL: https://mte-5854-show-international-date.solve.pr.rakr.net/solve

Login Url: https://mte-5854-show-international-date.solve.pr.rakr.net/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5854-show-international-date.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5854-show-international-date.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

**Comment by runrog** [comment]
Testing locally. 👍 LGTM

**Pull request merged**